src/component/Footer/Footer.js

```diff
@@ -2,7 +2,74 @@ import React from 'react';
 import './Footer.scss';
 
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="footerContainer">
```

Comment:
```suggestion
    <div className="footer">
```
* 최상위 className은 컴포넌트랑 이름 갖게 해 주시는게 위코드 컨벤션 입니다